gnosis/eth/tests/test_ethereum_client.py

```diff
@@ -848,7 +848,7 @@ def test_raw_batch_request(self, session_post_mock: MagicMock):
 
 class TestEthereumNetwork(EthereumTestCaseMixin, TestCase):
     def test_unknown_ethereum_network_name(self):
-        self.assertEqual(EthereumNetwork(2), EthereumNetwork.UNKNOWN)
+        self.assertEqual(EthereumNetwork(9), EthereumNetwork.UNKNOWN)
```

Comment:
9 Is ubiq.

Reply:
I think `UBIQ` is `8`

Reply:
From chainlist ubiq is 9, https://chainlist.org/chain/2